### PR TITLE
Add new rule to detect PST export when eDiscovery alert policy is disabled (M365)

### PR DESCRIPTION
**Pull request by nikitah4x**
Hey,
I've created a new rule to detect a PST export event when the "eDiscovery search or exported" alert policy is disabled.
An attacker can easily disable the alert policy, and you will not see it.
During my research, I found that when a user performs an export to a PST file using ExchangePowerShell or the Security & Compliance center, there is a log that can indicate an event when the alert policy is disabled.
Hunting for "New-ComplianceSearchAction" and payload containing '-Export' and 'pst,' you will have 100% detection.

### CODE REVIEW

File: rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+title: PST Export Alert Even When 'eDiscovery search or exported' Alert Policy is Disabled
+id: 6897cd82-6664-11ed-9022-0242ac120002
+status: experimental
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml:3]: Also the title is tool long, I think. It should be shorter. Leave the explanation to the description

[the remark this answers is not in the document]
Comment (nikitah4x) [reply rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml:3]: Hey, I've changed the title.

File: rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+title: PST Export Alert Using New-ComplianceSearchAction
+id: 6897cd82-6664-11ed-9022-0242ac120002
+related:
+    - id: 18b88d08-d73e-4f21-bc25-4b9892a4fdd0
+      type: similar
+status: experimental
+description: Alert when a user has performed an export to a search using 'New-ComplianceSearchAction' with the '-Export' flag. This detection will detect PST export even if the 'eDiscovery search or exported' alert is disabled in the O365.
+This rule will apply to ExchangePowerShell usage and from the cloud.
+references:
+    - https://attack.mitre.org/techniques/T1114/
```
Comment (nasbench) [inline rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml:10]:
```suggestion
```

File: rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - https://learn.microsoft.com/en-us/powershell/module/exchange/new-compliancesearchaction?view=exchange-ps
+author: 'Nikita Khalimonenkov'
```
Comment (nasbench) [inline rules/cloud/m365/microsoft365_pst_export_alert_when_policy_is_disabled.yml:12]:
```suggestion
author: Nikita Khalimonenkov
```

**Review by nasbench — CHANGES_REQUESTED** [review]
Looks good, only small changes.

Also please change the file name to `rules/cloud/m365/microsoft365_pst_export_alert_using_new_compliancesearchaction.yml`

**Comment by nikitah4x** [comment]
Done, changed file name, and resolved conversations.

**Comment by nikitah4x** [comment]
Fixed the filename, deleted by mistake the .yml

**Review by nasbench — APPROVED** [review]

**Pull request merged**